docs/compliance/generating_penalties.md

```diff
@@ -34,17 +42,22 @@ If you care about the data being sane, then use this method. It takes a little l
 - In the elicensing test environment, make a payment (or payments) that fully meet the obligation
 - Refresh the elicensing data in bciers
 - Note: If you want to test multiple payments over a period, update the received_date(s) of your payment records accordingly
-- Manually update elicensing_invoice.due_date to a date in the past
+- Manually update elicensing_invoice.due_date to a date in the past (note that the refresh service will overwrite this change with data from elicensing after 15 minutes on the next refresh, so make sure to do the next step immediately after changing the date)
```

Comment:
We talked about [turning off the refresh by setting the elicensing url to an empty string](https://teams.microsoft.com/l/message/19:81544b60877c4a5eb4fc7946ecbcb525@thread.tacv2/1754589584768?tenantId=6fdb5200-3d0d-4a8a-b036-d3685e359adc&groupId=29821293-3244-455d-beae-e01d27762cfe&parentMessageId=1754582336843&teamName=External%3A%20CAS%20-%20Clean%20Growth%20Digital%20Service%20Team&channelName=Developers&createdTime=1754589584768&ngc=true), but that doesn't work because we need to hit elicensing in the next step